src/common/components/Header/style.less

```diff
@@ -9,3 +9,48 @@
     line-height: @layout-header-height;
   }
 }
+
+.responsive-menu {
+  width: 100%;
+  padding-top: 0;
+  z-index: 3;
+
+  .ant-popover-content {
+    .ant-popover-arrow {
+      display: none;
+    }
+
+    .ant-popover-inner {
+      height: 100vh;
```

Comment:
we shouldnt use vh especially not for responsive styles since it doesnt work nicely on ios